source/_integrations/speedtestdotnet.markdown

```diff
@@ -22,7 +22,9 @@ The Speedtest.net integration uses the [Speedtest.net](https://speedtest.net/) w
 
 Most Speedtest.net servers require TCP port 8080 outbound to function. Without this port open you may experience significant delays or no results at all. See note on their [help page](https://www.speedtest.net/help).
 
-By default, a speed test will be run every hour. You can disable polling using system options and use the `update_entity` service to automate the speed test frequency.
+## Manual Polling
+
+By default, a speed test will be run every hour. To customize polling, you can disable automated polling. Go to {% my integrations title="**Settings** > **Devices & Services**" %}, and select the **Speedtest** integration. On the integration entry, select the three dots. Then, select **System options** and toggle the button to disable polling. To manually trigger a polling request, call the [`homeassistant.update_entity` service](/integrations/homeassistant/#service-homeassistantupdate_entity) as needed, either manually or via automations.
```

Comment:
This should be documented as something generic and not integration specific, as this is not integration-specific information.

I suggest reverting this PR for that reason.

/CC @c0ffeeca7

Reply:
Let me fix this, since I'm the one who merged it.